platforms/jvm/platform-jvm/src/main/java/org/gradle/jvm/tasks/Jar.java

```diff
@@ -109,23 +111,8 @@ private OutputChangeListener outputChangeListener() {
      * @since 2.14
      */
     @Override
-    @ToBeReplacedByLazyProperty
-    public String getMetadataCharset() {
-        return super.getMetadataCharset();
-    }
-
-    /**
-     * The character set used to encode JAR metadata like file names.
-     * Defaults to UTF-8.
-     * You can change this property but it is not recommended as JVMs expect JAR metadata to be encoded using UTF-8
-     *
-     * @param metadataCharset the character set used to encode JAR metadata like file names
-     * @since 2.14
-     */
-    @Override
-    public void setMetadataCharset(String metadataCharset) {
-        super.setMetadataCharset(metadataCharset);
-    }
+    @ReplacesEagerProperty
+    public abstract Property<String> getMetadataCharset();
```

Comment:
❓ I'm not sure this works as intended. This method only exists to change Javadoc AFAICT, is here a better way?

Reply:
Removing `@ReplacesEagerProperty` so the update machinery doesn't attempt anything here.